test/openAPI/features/wallet_binding.feature

```diff
@@ -0,0 +1,31 @@
+@method=POST @endpoint=/wallet-binding
+Feature: The endpoint to validate wallet create wallet user id
+
+  @smoke @unit @positive
+  Scenario: Successfully validates the wallet and generates the wallet user id smoke type test
+    Given Wants to validate the wallet and generate wallet user id
+    When Send POST /wallet-binding request with given "n3fy2qkg9r7h2" as individualId and "OTP" as authFactorType and "encoded-json" as format and "string" as challenge and publicKey
+    Then Receive a response from the /wallet-binding endpoint
+    And The /wallet-binding response should be returned in a timely manner 15000ms
+    And The /wallet-binding endpoint response should have status 200
+    And The /wallet-binding endpoint response should match json schema with no errors
+
```

Comment:
can we also add one more condition that expireDateTime should be in the future date ?

Reply:
yes, added.

Reply:
I see we have a check that it should not be empty, but we want to check if the date give is in the future and not in past

Reply:
Ok, I understand what you mean. 
I'm just wondering how we can check it because the `expireDateTime` field has string type, so we are not sure in what format we can received it (we can also received "11:10, 20.07.2023" - just silly example). 
In my opinion the best solution is that `expireDateTime` should has ISO Date Time Format yyyy-MM-dd'T'HH:mm:ss.SSSXXX because there is also information about timezone. Or there should be only date but also we need to know what format should be used.
Like that we can easily test it and be sure that we compared this date well.
What do you thing about it @vishwa-vyom ?

Reply:
Yes, you are right. The date time format is standardized to ensure interoperability between systems. It should follow ISO 8601 format (yyyy-MM-dd'T'HH:mm:ss.SSSZ) eg.2023-07-11T09:40:09.516Z. Some areas of API docs also documents this

https://github.com/GovStackWorkingGroup/bb-identity/blob/1.0-QA/api/Identity-Provider.yaml#L989

I think we should also check for this format in all the responses wherever datetime fields are used.

Reply:
Ok great, I will add test steps to check it ;)